kshebeko/lesson4/src/com/company/kshebeko_class/GardenFlower.java

```diff
@@ -0,0 +1,30 @@
+package com.company.kshebeko_class;
+
+public class GardenFlower extends Flower {
+    public int stemLength = 50;
+    public String flowerBud;
+
+    public GardenFlower(int stemLength) { //Конструктор
+        this.stemLength = stemLength;
+    }
+
+    public GardenFlower(String flowerBud) { //Конструктор
+        this.flowerBud = flowerBud;
+    }
+
+    public GardenFlower(String name, int price) { //Конструктор
```

Comment:
comments are not needed here, it's quiet clear that these methods are constructors